sdk/cognitivelanguage/azure-ai-language-questionanswering/README.md

```diff
@@ -0,0 +1,261 @@
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/azure-sdk-for-python.client?branchName=main)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46?branchName=main)
+
+# Azure Cognitive Language Services Question Answering client library for Python
+
+Question Answering is a cloud-based API service that lets you create a conversational question-and-answer layer over your existing data. Use it to build a knowledge base by extracting questions and answers from your semi-structured content, including FAQ, manuals, and documents. Answer users’ questions with the best answers from the QnAs in your knowledge base—automatically. Your knowledge base gets smarter, too, as it continually learns from user behavior.
+
+[Source code][questionanswering_client_src] | [Package (PyPI)][questionanswering_pypi_package] | [API reference documentation][questionanswering_refdocs] | [Product documentation][questionanswering_docs] | [Samples][questionanswering_samples]
+
+## Getting started
+
+### Prerequisites
+
+* Python 2.7, or 3.6 or later is required to use this package.
+* An [Azure subscription][azure_subscription]
+* An existing Question Answering resource
+
+> Note: the new unified Cognitive Language Services are not currently available for deployment.
+
+### Install the package
+
+Install the Azure QuestionAnswering client library for Python with [pip][pip_link]:
+
+```bash
+pip install azure-ai-language-questionanswering --pre
```

Comment:
tiny nit: you don't need `--pre` yet, bc there hasn't been a GA release of QnA yet